### PR TITLE
Feature/Last Login

### PR DESCRIPTION
**Pull request by mblomdahl**
Adds `modified_at` + `last_login_at` from #78. The `created_at` might be rather convoluted, the way we initially envisioned it... Let's defer that until we've done some thinking. :)

### CODE REVIEW

File: tests/models/test_collection.py [hunk 1]
```diff
@@ -36,6 +36,23 @@ def test_created_at_defaults_to_datetime():
...
 
 @pytest.mark.usefixtures('db')
+def test_modified_at_defaults_to_current_datetime():
+    """Test modified date."""
+    collection = Collection('KBU', 'Outdated name', 'library')
+    collection.save()
+    first_modified_at = collection.modified_at.isoformat()
+
+    # Initial 'modified_at' matches 'created_at' (at least at the minute level).
+    assert first_modified_at[:16] == collection.created_at.isoformat()[:16]
```
Comment (mxtthias) [inline tests/models/test_collection.py]: Use `assert (datetime.utcnow() - updated_user.last_login_at).total_seconds() < 10` instead?

File: tests/models/test_permission.py [hunk 2]
```diff
@@ -36,6 +36,23 @@ def test_created_at_defaults_to_datetime(user, collection):
...
 
 @pytest.mark.usefixtures('db')
+def test_modified_at_defaults_to_current_datetime(user, collection):
+    """Test modified date."""
+    permission = Permission(user=user, collection=collection)
+    permission.save()
+    first_modified_at = permission.modified_at.isoformat()
+
+    # Initial 'modified_at' matches 'created_at' (at least at the minute level).
+    assert first_modified_at[:16] == permission.created_at.isoformat()[:16]
```
Comment (mxtthias) [inline tests/models/test_permission.py]: Use `assert (datetime.utcnow() - updated_user.last_login_at).total_seconds() < 10` instead?

File: tests/models/test_user.py [hunk 3]
```diff
@@ -33,6 +33,23 @@ def test_created_at_defaults_to_datetime():
...
 
 @pytest.mark.usefixtures('db')
+def test_modified_at_defaults_to_current_datetime():
+    """Test modified date."""
+    user = User('foo@kb.se', 'Wrong Name')
+    user.save()
+    first_modified_at = user.modified_at.isoformat()
+
+    # Initial 'modified_at' matches 'created_at' (at least at the minute level).
+    assert first_modified_at[:16] == user.created_at.isoformat()[:16]
```
Comment (mxtthias) [inline tests/models/test_user.py]: Use `assert (datetime.utcnow() - updated_user.last_login_at).total_seconds() < 10` instead?

File: tests/models/test_permission.py (hunk 2 above)
Comment (mblomdahl) [reply tests/models/test_permission.py]: Fixed

File: tests/models/test_collection.py (hunk 1 above)
Comment (mblomdahl) [reply tests/models/test_collection.py]: Fixed

File: tests/models/test_user.py (hunk 3 above)
Comment (mblomdahl) [reply tests/models/test_user.py]: Fixed

**Pull request merged**